src/components/LayersPanel/LayersPanel.js

```diff
@@ -27,6 +27,25 @@ class LayersPanel extends React.PureComponent {
       window.docViewer.updateView();
     }
   }
+  componentDidMount() {
+    core.getDocument().on('layersUpdated', this.onLayersUpdated);
```

Comment:
It doesn't seem like this works as expected. componentDidMount is fired when the I actually open the panel so if there are any programmatic layer changes before the panel is opened then they won't be updated here.

Maybe we should be listening for this event somewhere else globally and calling the action there. Then in this component we'll just use the selector which also means we shouldn't need to use a forceUpdate?